docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss

```diff
@@ -148,7 +148,7 @@ caption:not([class]) {
   font-size: hb-calculate-rems(14px);
 
   @include hb-colorful {
-    @include hb-global-color('color', 'gray');
+    @include hb-global-color("color", "gray-dark");
```

Comment:
I fixed it here since I reviewed other captions and they were having contrast issues

Reply:
Looks good, that makes sense! Can you just please change the double quotes (`"`) to single quotes (`'`) for consistency?